sw/airborne/modules/orange_avoider/orange_avoider.h

```diff
@@ -13,13 +13,16 @@
 #ifndef ORANGE_AVOIDER_H
 #define ORANGE_AVOIDER_H
 #include <inttypes.h>
+#include "state.h"
```

Comment:
do you really have to add state here? don't you just need math/pprz_geodetic_int.h for EnuCoor_i